ledger-tool/src/main.rs

```diff
@@ -1238,6 +1243,18 @@ fn load_bank_forks(
             accounts_update_notifier,
             &Arc::default(),
         );
+    let replaying_backend = arg_matches
+        .value_of("replaying_backend")
+        .map(ReplayingBackend::from)
+        .unwrap();
+    let banking_backend = arg_matches
```

Comment:
ledger-tool doesn't do banking. can we drop this?

Reply:
+1

Reply:
nice catch; but it will do so soon (tm) with `solana-ledger-tool simulate-leader-blocks` subcommand with bankingtrace files.

I've just left here for easier revewing. Commenting-out or removing altogether can be option.

Reply:
had a feeling it was for the simulate, but think we should drop it until it's added. (similarly with the `UnifiedScheduler` variant 😄

Reply:
removed as this pr's nearing merging: 296b90908bbab2b25d0445a3132abce74cc9801d